pyronear/datasets/wildfire/wildfire.py

```diff
@@ -45,9 +46,15 @@ def __init__(self, metadata, path_to_frames, target_names=None, transform=None):
 
         # default target is fire detection (0/1)
         self.target_names = target_names or ['fire']
-        self.path_to_frames = path_to_frames
         self.transform = transform
 
+        # converting path_to_frames to path type
+
+        if isinstance(path_to_frames, PurePath):
```

Comment:
Python `Path` object is idempotent over strings, so you can directly use it either on strings or paths:
```python
path_str = "/path/to/lovely/directory"
path_pure = Path("/path/to/lovely/directory")

assert Path(path_str) == path_pure # True
assert Path(path_pure) == path_pure # True (idempotence)
```

Reply:
Ok then the if statement isn't relevant here, will simplify applying Path directly on input path_to_frames.